vnext/react-native.config.js

```diff
@@ -2,16 +2,77 @@
 require('source-map-support').install();
 
 const cli = require('@react-native-windows/cli');
+const { execSync } = require('child_process');
+const path = require('path');
+
+const cliDir = path.dirname(require.resolve('@react-native-community/cli/package.json'));
+const execaPath = require.resolve('execa', { paths: [cliDir] });
+const execa = require(execaPath);
 
 module.exports = {
-    // **** This section defined commands and options on how to provide the Windows platform to external applications
-    commands: cli.commands,
-    platforms: {
-      windows: {
-        linkConfig: () => null,
-        projectConfig: cli.projectConfig,
-        dependencyConfig: cli.dependencyConfig,
-        npmPackageName: 'react-native-windows',
-      },
+  // **** This section defined commands and options on how to provide the Windows platform to external applications
+  commands: cli.commands,
+  platforms: {
+    windows: {
+      linkConfig: () => null,
+      projectConfig: cli.projectConfig,
+      dependencyConfig: cli.dependencyConfig,
+      npmPackageName: 'react-native-windows',
     },
-  };
+  }
+};
+
+// All our health checks are windows only...
+if (process.platform === 'win32') {
+  const rnwDepScriptPath = path.resolve(__dirname, 'Scripts/rnw-dependencies.ps1');
+  try {
+    const rnwDeps = execSync(`powershell ${rnwDepScriptPath} -NoPrompt -ListChecks`);
```

Comment:
How long does this take? Would be great if we could avoid blocking CLI init on synchronously running another script.

Reply:
That one is pretty quick since it really doesn't do much.  The PS script with the ListChecks flag basically just zips through an array of checks and prints out some basic info about them.